dask_jobqueue/pbs.py

```diff
@@ -60,66 +62,43 @@ class PBSCluster(JobQueueCluster):
 
     >>> cluster.adapt()
     """
+
+    #Override class variables
+    submit_command = 'qsub'
+    cancel_command = 'qdel'
+
     def __init__(self,
-                 name='dask',
-                 queue='regular',
+                 name='dask-worker',
+                 queue=None,
                  project=None,
-                 resource_spec='select=1:ncpus=36:mem=109GB',
-                 threads_per_worker=4,
-                 processes=9,
-                 memory='7GB',
+                 resource_spec='select=1:ncpus=24:mem=100GB',
                  walltime='00:30:00',
-                 interface=None,
-                 death_timeout=60,
-                 extra='',
+                 job_extra=[],
+                 local_directory='$TMPDIR',
```

Comment:
I have a slight preference to set this to None by default. Either that or `os.getenv('TMPDIR', None)`.

Reply:
I believed that $TMPDIR was a PBS default, but apparently it should be PBS_TMPDIR. Can you check on the cluster you are using?
Maybe it is more generic to just remove this arg by default.

Reply:
On our system (NCAR's cheyenne), `TMPDIR` is set to `/var/tmp/{JOB_ID}`.  So given the divergence, I suggest just removing this as a default.

Reply:
Then our systems are configured with the same variable (if not set to the same directory), it is difficult to generalise from two examples, but TMPDIR seems to be used for a scratch dir for you and me. Stilled removed it maybe I will add a comment for this.